src/GitHubCreateMergePRs/config.xml

```diff
@@ -15,10 +15,13 @@
     <merge from="release/dev17.2" to="release/dev17.3" />
 
     <!-- Roslyn release branches (flowing between releases) -->
-    <merge from="release/dev17.3" to="main" />
+    <merge from="release/dev17.3" to="release/dev17.4" />
+    <merge from="release/dev17.3-vs-deps" to="release/dev17.4-vs-deps" />
+    <merge from="release/dev17.4" to="main" />
 
```

Comment:
Looks like we need a dev17.4-vs-deps to main-vs-deps merge until we are able to remove -vs-deps branches.